src/sentry/api/helpers/group_index/update.py

```diff
@@ -591,7 +294,7 @@ def update_groups(
             else:
                 result["statusDetails"] = {}
         if group_list and status_updated:
-            activity_type, activity_data = handle_status_update(
+            handle_status_update(
```

Comment:
I analyzed the code from a couple of weeks ago, making sure I did not regress it. However, `activity_type` and `activity_data` are not used from this point on in the code:


https://github.com/getsentry/sentry/blob/12d04478fab99de1787df334ce77d36632fda07d/src/sentry/api/helpers/group_index/update.py#L610-L621
https://github.com/getsentry/sentry/blob/58657d1060b3e67a76b40867ca7474aa7b0bd19b/src/sentry/issues/status_change.py#L144-L150

It seems that activity is read back from the DB here, thus, no need to return it as variables:
https://github.com/getsentry/sentry/blob/12d04478fab99de1787df334ce77d36632fda07d/src/sentry/api/helpers/group_index/update.py#L632-L637